tests/conftest.py

```diff
@@ -1,9 +1,10 @@
 import pytest
+from cdisutilstest.code.conftest import indexd_server
+from indexclient.client import IndexClient
 
-from cdisutilstest.code.conftest import indexd_client, indexd_server
 
```

Comment:
just import the indexd_client fixture in cdisutilstest and use that, don't need to make another index_client?

Reply:
Definitely, a fixtures whole point is to not duplicate test code. Make index_client take indexd_client as an argument and return the indexd_client, while functionally scoped. Keeps indexclient's test code consistent, and whose to know how many different index clients may ever exist.
The cdisutils pin is 0.2.2 [here](https://github.com/uc-cdis/cdisutils-test/releases)
https://github.com/uc-cdis/cdisutils-test/blob/02d69e5e9be4305717674dcc9efb2055234d08e0/cdisutilstest/code/conftest.py#L34

Reply:
@jbarno I made changes to index_client which is not on the one I will import, cos the one cdisutil does not have my changes.